common/changes/@itwin/viewer-react/prep-browser-authorization-1.0_2023-03-22-20-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Support @itwin/browser-authorization@0.8-1.0",
+      "type": "none"
+    }
+  ],
+  "packageName": "@itwin/viewer-react"
```

Comment:
did this pkg have a dep on browser-auth?

Reply:
Nope thanks good catch

Reply:
fixed